src/Jackett.Common/Definitions/torrentfactory.yml

```diff
@@ -0,0 +1,176 @@
+      - {id: 55 , cat: Books/Ebook, desc: "Ebooks - LIBROS"}
+      - {id: 63 , cat: Books/Magazines, desc: "Ebooks - Publicacion Semanal"}
+      - {id: 64 , cat: Books/Magazines, desc: "Ebooks - Prensa Diaria"}
+      - {id: 79 , cat: Books/Other, desc: "Ebooks - Adultos Ebooks"}
+      - {id: 83 , cat: Books/Magazines, desc: "Ebooks - Publicacion Quincena"}
+      - {id: 88 , cat: Books/Magazines, desc: "Ebooks - Publicacion Mensual"}
+      - {id: 97 , cat: Books/Comics, desc: "Ebooks - Comic eBooks"}
+
+    modes:
+      search: [q]
+      tv-search: [q, season, ep]
+      movie-search: [q]
+
+  settings:
+    - name: username
+      type: text
+      label: Username
+    - name: password
+      type: password
+      label: Password
+    - name: sort
+      type: select
+      label: Sort requested from site
+      default: "3"
+      options:
+        "3": "created"
+        "5": "seeders"
+        "4": "size"
+        "2": "title"
+    - name: type
+      type: select
+      label: Order requested from site
+      default: "2"
+      options:
+        "2": "desc"
+        "1": "asc"
+    - name: info
+      type: info
+      label: Results Per Page
+      default: For best results, change the <b>Torrents per page:</b> setting to <b>100</b> on your account profile.
+
+  login:
+    path: index.php?page=login
+    method: post
+    inputs:
+      uid: "{{ .Config.username }}"
+      pwd: "{{ .Config.password }}"
+    error:
+      - selector: td.lista span[style="color:#FF0000;"]
+    test:
+      path: index.php
+
+  search:
+    path: index.php
+    inputs:
+      page: torrents
+      $raw: "&category={{ range .Categories }}{{.}};{{end}}"
+      active: 1
+      search: "{{ .Keywords }}"
+      order: "{{ .Config.sort }}"
+      by: "{{ .Config.type }}"
+
+    rows:
+      selector: "#Mcol > div > div.block-content > div > div > div > table > tbody > tr:nth-child(3) > td > table tr:not(:first-child)"
+
+    fields:
+      category:
+        selector: td:nth-child(1) a
+        attribute: href
+        filters:
+          - name: querystring
+            args: category
+      title:
+        selector: td[valign="middle"] a
+      details:
+        selector: td[valign="middle"] a
+        attribute: href
+      banner:
+        optional: true
+        selector: td[valign="middle"] a
+        attribute: onmouseover
+        filters:
+          - name: regexp
+            args: "src=(.+?) "
+      size:
+        selector: td:nth-child(9)
+      seeders:
+        selector: td:nth-child(5) a
+      leechers:
+        selector: td:nth-child(6) a
+      grabs:
+        text: 0
+      grabs:
+        selector: td:nth-child(7) a
+        optional: true
+      date:
+        selector: td:nth-child(4)
+        filters:
+          - name: dateparse
+            args: "02/01/2006"
+      download:
+        selector: a[href^="download.php"]
+        attribute: href
+      downloadvolumefactor:
+        case:
+          img[src$="gold.gif"]: 0.05
```

Comment:
very unusual DLVF values.
typically gold is 0 (100% free), silver is 0.5 (50% free) and bronze is 0.75 (25% free)
are you confident you have these correct?

Reply:
I think I did it right
![image](https://user-images.githubusercontent.com/10577978/74103612-c9e4c700-4b4d-11ea-80ce-9185be835264.png)